tests/unit/cli/test_parser.py

```diff
@@ -46,78 +45,6 @@ def test_get_headers_with_value_error(mock_get_headers):
     mock_get_headers.assert_called_once()
 
 
-@patch(
-    "geneweaver.client.parser.general.get_headers",
-    return_value=(["header1", "header2", "header3"], 0),
-)
-@patch(
-    "geneweaver.client.parser.general.data_file_to_dict_n_rows",
-    return_value=[{"header1": "val1", "header2": "val2", "header3": "val3"}],
-)
-def test_preview(mock_data_file_to_dict_n_rows, mock_get_headers):
-    """Test the preview CLI command."""
-    # Simulate the CLI execution
-    result = runner.invoke(cli, ["utils", "preview", "fake_path"])
-
-    print(result.output)
-
-    # # Check the output message
-    assert "header1" in result.output
-    assert "header2" in result.output
-    assert "header3" in result.output
-    assert "val1" in result.output
-    assert "val2" in result.output
-    assert "val3" in result.output
-
-    # # Check the exit code
-    assert result.exit_code == 0
-
-
-@patch(
-    "geneweaver.client.parser.general.get_headers",
-    return_value=(["header1", "header2", "header3"], 0),
-)
-@patch(
-    "geneweaver.client.parser.general.data_file_to_dict_n_rows",
-    return_value=[{"header1": "val1", "header2": "val2", "header3": "val3"}],
-    side_effect=EmptyFileError("File is empty."),
-)
-def test_preview_get_data_error(mock_data_file_to_dict_n_rows, mock_get_headers):
-    """Test the preview CLI command with an error."""
-    # Simulate the CLI execution
-    result = runner.invoke(cli, ["utils", "preview", "fake_path"])
-
-    # Check the output message
-    assert "File is empty." in result.output
-    # Check the exit code
-    assert result.exit_code == 1
-
-    # Check if the mocked function was called
-    mock_get_headers.assert_called_once()
-    mock_data_file_to_dict_n_rows.assert_called_once()
-
-
-@patch(
-    "geneweaver.client.parser.general.get_headers",
-    return_value=(["header1", "header2", "header3"], 0),
-    side_effect=ValueError("File is empty."),
```

Comment:
I thought you changed the error from File is empty to Something went wrong...

Reply:
I did, and then I removed the tests for this entirely. I decided that since these commands are still "alpha", they are subject to change. I will write comprehensive tests for them when they move to "beta".

Almost all of the logic involved in processing the data is tested in the geneweaver.core library anyway.